R/datasets.R

```diff
@@ -1,5 +1,25 @@
 #' Retrieve a Dataset from the `admiraldev_environment` environment
```

Comment:
```suggestion
#' Retrieve a Dataset from the `admiraldev_environment()` environment
```

Does this become linkable to the docs for it??  It isn't a function, but it has roxygen style...wondering...